### PR TITLE
#1-Added NavBar along with background image

### PR DESCRIPTION
**Pull request by himanshu-dhiman**

### CODE REVIEW

File: public/wp-content/themes/ColoredCow/header.php [hunk 1]
```diff
@@ -6,8 +6,24 @@
...
     <title><?php echo bloginfo('name'); ?></title>
-    <?php wp_head(); ?>
+    <?php wp_head();?>
 </head>
 
+	<nav class=" navbar fixed-top navbar-toggleable-md navbar-inverse bg-inverse">				
+		<button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+		</button>
+		<a class="navbar-brand" href="#" style="font-family: ;"><img src="<?php  $image_url=get_template_directory_uri()."/dist/img/logo.png"; echo esc_url($image_url);?>" width="180" height="45" class="d-inline-block align-top" alt=""></a>
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/header.php]: esc_url(get_template_directory_uri()."/dist/img/logo.png")

Also see formatting

File: public/wp-content/themes/ColoredCow/header.php [hunk 2]
```diff
@@ -6,8 +6,24 @@
...
+		<div class="collapse navbar-collapse justify-content-end" id="navbarSupportedContent">
+			<ul class="navbar-nav">
+				<li class="nav-item active">
+					<a role="button" class="btn btn-outline-secondary" href="wp/wp-login.php" style="font-family: 'Oswald'; font-size: 17px;">
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/header.php]: use external css.

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 3]
```diff
@@ -33,24 +33,19 @@ Text Domain: coloredcow
...
-h5{
-	font-family: 'Marcellus SC'; 
-	font-size: 85px; 
-	color:$goldenrod;
-}
-.lead{
-	font-size: 20px; 
-	font-family: 'Open Sans';
-	padding-bottom: 5px;
+#footer {
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: prefer classes

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 4]
```diff
@@ -33,24 +33,19 @@ Text Domain: coloredcow
...
+	text-align:center;
+	bottom: 0;
+	width: 100%;
+	height:60px;
+    background-color: black;
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: use variables.scss

**Review by Shubham2301 — CHANGES_REQUESTED** [review]

File: public/wp-content/themes/ColoredCow/functions.php [hunk 5]
```diff
@@ -2,12 +2,12 @@
 
 if ( ! function_exists( 'cc_scripts' ) ) {
     function cc_scripts() {
+        
+        wp_enqueue_script('cc-jquery-script','https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js');
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/functions.php]: please use WP's inbuilt jquery.

File: public/wp-content/themes/ColoredCow/functions.php [hunk 6]
```diff
@@ -2,12 +2,12 @@
...
         wp_enqueue_script('cc-bootstrap-script','https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js');
-
         wp_enqueue_script('cc-fontawesome-icons','https://use.fontawesome.com/ffc2c94a85.js');
-        wp_enqueue_script('cc-jquery-script','https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js');
-        wp_localize_script( 'main', 'PARAMS', array('ajaxurl' => admin_url('admin-ajax.php')) );
 
+        wp_localize_script( 'main', 'PARAMS', array('ajaxurl' => admin_url('admin-ajax.php')) );
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/functions.php]: what's the purpose of putting it here? Does it make sense to use it here? Read the documentation for wp_localize_script please.

File: public/wp-content/themes/ColoredCow/header.php (hunk 1 above)
Comment (rathorevaibhav) [reply public/wp-content/themes/ColoredCow/header.php]: Also, let's use the height/width in stylesheet.

Comment (rathorevaibhav) [reply public/wp-content/themes/ColoredCow/header.php]: alt tag should not be empty.

Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/header.php]: you can directly echo the complete URL without creating a variable. Also, it's not preferred to have multiple statements in an inline PHP tag.

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 7]
```diff
@@ -40,12 +40,12 @@ Text Domain: coloredcow
...
-	width: 100%;
-	height:60px;
-    background-color: black;
-    color: white;
+.logo{
+	width:180px;
+	height:45px;
+}
+.admin-button {
+	font-family: 'Oswald';
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: use fonts.scss

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 8]
```diff
@@ -33,24 +33,19 @@ Text Domain: coloredcow
 		display: block;
 	}
 }
+.btn:hover
+{
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: the bracket should be in line 36. Please use same format throughout.

File: public/wp-content/themes/ColoredCow/src/scss/style.scss (hunk 4 above)
Comment (rathorevaibhav) [reply public/wp-content/themes/ColoredCow/src/scss/style.scss]: Indentation.

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 9]
```diff
@@ -33,24 +33,19 @@ Text Domain: coloredcow
...
-	font-size: 20px; 
-	font-family: 'Open Sans';
-	padding-bottom: 5px;
+#footer {
+	text-align:center;
+	bottom: 0;
+	width: 100%;
+	height:60px;
+    background-color: black;
+    color: white;
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: should be a variable.

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]

**Pull request merged**